### ISSUE TITLE
lxqt-qtplugin don't pick up qt private headers reliable

### ISSUE THREAD

**Issue by agaida** [lxde/lxqt#1222]
this is a known Qt-Bug https://bugreports.qt.io/browse/QTBUG-57060 - we should add a dummy function to the qtplugins that explicitly use one of the "not so private" Symbols and give the linker a chance

Found in Debian, Ubuntu, Parabola GNU/Linux so far

**Comment by agaida** [comment]
it seems that i was wrong and i run into a kernel bug with overlay file systems - will test this with a fixed kernel

**Comment by tsujan** [comment]
Just curious: which kernel version is it?

**Comment by agaida** [comment]
the faulty kernel was 4.9.0-towo.1-siduction-amd64 #1 SMP PREEMPT siduction 4.9-1 (2016-12-13) x86_64 GNU/Linux and i guess the 4.8 series, same kernel with this patch: https://launchpadlibrarian.net/149319334/overlayfs_inotify.patch applied works like charm

**Comment by agaida** [comment]
The fine (or at whish not so fine) thing is: The behaviour is reproducable with the current debian kernel from sid, just built an iso and checked that.

**Comment by tsujan** [comment]
I don't know about sid kernels but I tried 4.9.0 on Manjaro. At first everything seemed OK but, after a few hours, pcmanfm-qt crashed for no reason and didn't restart, I couldn't run any app from the main menu or by using keyboard shortcuts, I couldn't even log out -- only ctrl+alt+f2 worked and I was able to shut down and boot into the good LTS kernel 4.4.39. It wasn't easy because kernel 4.9.0 hung somewhere during that shutdown but, fortunately, the partitions were unmounted.

I'd never encountered such a problem with a kernel -- not even Zen kernels.
